python/demo/demo_lagrange_variants.py

```diff
@@ -163,15 +163,15 @@ def saw_tooth(x):
 # elements, and plot the finite element interpolation.
 
 # +
-mesh = mesh.create_unit_interval(MPI.COMM_WORLD, 10)
+msh = mesh.create_unit_interval(MPI.COMM_WORLD, 10)
```

Comment:
I do personally not like the variable name `msh`, as it to me seems to like the variable to the GMSH MSH format (https://gmsh.info/doc/texinfo/gmsh.html#MSH-file-format). Could we instead use `domain`? Then usage in `ufl.Measure` would be, `ufl.Measure("dx", domain=domain, ...)`.

Reply:
I think `msh` is the best compromise.